src/sentry/features/temporary.py

```diff
@@ -46,6 +46,7 @@ def register_temporary_features(manager: FeatureManager):
     manager.add("organizations:anr-analyze-frames", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
     manager.add("organizations:anr-improvements", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
     manager.add("organizations:api-auth-provider", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
+    manager.add("organizations:api-id-or-slug-support", OrganizationFeature, FeatureHandlerStrategy.REMOTE)
```

Comment:
should we roll this out on flagr or some other method?

Reply:
We will be using an option now!